test/integration/cli/test/index.test.js

```diff
@@ -394,6 +394,30 @@ describe('CLI Usage', () => {
         'Invalid project directory provided, no such directory'
       )
     })
+
+    test('should validate keepAliveTimeout argument', async () => {
```

Comment:
Added these integration tests but locally they hang. Not sure why.

Reply:
@styfle can you approve the workflows so we can see if these fail in the ci? Or do you have some insight as to why they're hanging when ran locally?

Reply:
This test was unnecessary because the `args` module seems to already filter for `NaN`s.